src/main/java/org/broadinstitute/consent/http/db/ConsentDAO.java

```diff
@@ -18,39 +18,46 @@
 @RegisterRowMapper(ConsentMapper.class)
 public interface ConsentDAO extends Transactional<ConsentDAO> {
 
-    @SqlQuery("select * from consents where consentId = :consentId and active=true")
+    /**
+     * Find consent by id
+     *
+     * @return Consent
+     */
+    @SqlQuery("SELECT * FROM consents WHERE consent_id = :consentId AND active=true")
     Consent findConsentById(@Bind("consentId") String consentId);
 
     @SqlQuery(
-        " SELECT c.* "
-            + " FROM consents c "
-            + " INNER JOIN consentassociations cs ON c.consentid = cs.consentid "
-            + " WHERE cs.datasetid = :datasetId")
+            " SELECT c.* "
+                    + " FROM consents c "
+                    + " INNER JOIN consent_associations ca ON c.consent_id = ca.consent_id "
+                    + " WHERE ca.dataset_id = :datasetId")
     Consent findConsentFromDatasetID(@Bind("datasetId") Integer datasetId);
```

Comment:
This has 1 usage in a test ... we can remove this.

Reply:
Removed **findConsentNameFromDatasetID** and the associated test - I think that's what you meant for this comment. findConsentFromDatasetID had 4 references in ConsentService, DatasetService, and ElectionService.

Reply:
Thank you, yes, you read my mind.